docs/DeveloperGuide.adoc

```diff
@@ -620,6 +620,84 @@ image::HideInfoSequenceDiagrams.png[]
 
 The censored data are only hidden from the window, not changed in the database. The `BrowserPanel` on the bottom right of the window will still show all the information including the hidden information.
 
+=== Send Reminder Email feature
+
+==== Current Implementation
+
+The email sending mechanism is supported by the https://docs.oracle.com/javaee/7/api/javax/mail/package-summary.html[`JavaMail`] library. This feature is composed of three commands: `check`, `setemail` and `remind`.
+
+* `check` command: display the email address that the user has set to send emails from. The app will censor the email address.
+[NOTE]
+If user has not set an email, the email address will be a string `"default"` and the app will throw a `CommandException`.
```

Comment:
How is the `CommandException` handled? What will the user see?

Reply:
This exception will handled in ResultDisplay class and the ResultDisplay window will show the message contained in this exception. Should I add this sentence or I just write like "the app will show a have not set an email message"?